src/test/java/org/hydev/mcpm/utils/PairTest.java

```diff
@@ -0,0 +1,39 @@
+package org.hydev.mcpm.utils;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+/**
+ * Tests for Pair class
+ */
+class PairTest
+{
+    @Test
+    void gettersSetters()
+    {
+        var p = new Pair<>("a", 1);
+
+        assert p.k().equals(p.getKey());
```

Comment:
Regular assert again. This is perfect for `assertEquals`!

Reply:
ok replace `(assert )(.*?)(.equals\()(.*)\);` with `assertEquals($2, $4);`